index.js

```diff
@@ -11,34 +12,36 @@ dotenv.config();
 
 // Get user input -
 
-// TODO: validate user input
-let searchName = (await inquirer.prompt({
-  name: 'search_name',
-  type: 'input',
-  message: 'What do you want to search in google?', 
-})).search_name;
-
-// TODO: validate user input
-let searchNumber = (await inquirer.prompt({
-  name: 'search_number',
-  type: 'input',
-  message: 'How many search results do you want? (max 100)', 
-})).search_number;
-
-// Lets do some magic - API call
-const search = `q=${searchName}&num=${searchNumber}`;
-
-const response = await (await fetch(`https://google-search3.p.rapidapi.com/api/v1/search/${search}`, {
-  method: 'GET',
-  headers: {
-    'x-rapidapi-key': process.env.API_KEY,
-    'x-proxy-location': 'US',
-  },
-})).json();
-
-// toModel
-const searchItems = await response.results.map((item) => new SearchItem(item.title, item.description, item.link));
-
-// Create excel!
-const arrayOfArray = excelUtils.toExcelArrayOfArray(searchItems);
-excelUtils.createExcel(arrayOfArray, searchName);
+// // TODO: validate user input
+// let searchName = (await inquirer.prompt({
+//   name: 'search_name',
+//   type: 'input',
+//   message: 'What do you want to search in google?', 
+// })).search_name;
+
+// // TODO: validate user input
+// let searchNumber = (await inquirer.prompt({
+//   name: 'search_number',
+//   type: 'input',
+//   message: 'How many search results do you want? (max 100)', 
+// })).search_number;
+
+// // Lets do some magic - API call
+// const search = `q=${searchName}&num=${searchNumber}`;
+
+export async function createExcel(searchName, searchNumber) {
+  const response = await (await fetch(`https://google-search3.p.rapidapi.com/api/v1/search/q=${searchName}&num=${searchNumber}`, {
+    method: 'GET',
+    headers: {
+      'x-rapidapi-key': process.env.API_KEY,
+      'x-proxy-location': 'US',
+    },
+  })).json();
+
+  // toModel
+  const searchItems = await response.results.map((item) => new SearchItem(item.title, item.description, item.link));
+
+  // Create excel!
+  const arrayOfArray = excelUtils.toExcelArrayOfArray(searchItems);
+  excelUtils.createExcel(arrayOfArray, searchName);
```

Comment:
is this the original CLI that we made ? if so, why are u modifying this ?

BTW, ideally I want u to make a js file that ur sharing between the 2 (the react app n this cli script), but we can do that in another step. in the meantime, any reason ur modifying this already existing one ?

Reply:
I was just testing some stuff and forgot to revert the file haha, the sharing thing I tried that first but some stupid react errors kept occuring

Reply:
okok we'll do it as part of the next iteration, perhaps we can publish an npm package so u get familiar with that process (creating libs n etc)